llvm/lib/CodeGen/SelectionDAG/SelectionDAG.cpp

```diff
@@ -1155,11 +1155,11 @@ void SelectionDAG::DeallocateNode(SDNode *N) {
 
 #ifndef NDEBUG
 /// VerifySDNode - Check the given SDNode.  Aborts if it is invalid.
-static void VerifySDNode(SDNode *N, const TargetLowering *TLI) {
+static void VerifySDNode(const SelectionDAG &DAG, SDNode *N) {
```

Comment:
could we simply make VerifySDNode a member function?

Reply:
Made it a member function. Not sure if it should be guarded by `LLVM_ENABLE_ABI_BREAKING_CHECKS`. It doesn't add new data members / virtual functions, so this change shouldn't be an ABI breaking, right?